app/routes/__init__.py

```diff
@@ -209,5 +209,5 @@
 api.add_resource(ProjectUsersHandleInviteView,
                  '/projects/<string:project_id>/users/handle_invite')
 # system status
-api.add_resource(SystemStatusView, '/system_status')
-api.add_resource(SystemSummaryView, '/system_summary')
```

Comment:
This should be there since we shall need it to route to the `SystemSummaryView` controller or else it will through a 404 error when the `/system_summary` is hit or accessed.